packages/ra-core/src/i18n/useTranslate.ts

```diff
@@ -1,6 +1,6 @@
 import { useCallback } from 'react';
 
-import { Translate } from '../types';
+import { TranslateFunction } from '../types';
```

Comment:
This is technically a BC break and it should appear front and foremost in the PR description